src/main/java/de/qabel/desktop/ui/actionlog/ActionlogController.java

```diff
@@ -104,14 +89,14 @@ protected void handleSubmitButtonAction() throws QblDropPayloadSizeException, En
 
 	void sendDropMessage(Contact c, String text) throws QblDropPayloadSizeException, QblNetworkInvalidResponseException, PersistenceException {
 		DropMessage d = new DropMessage(identity, text, "dropMessage");
-		dropMessageRepository.addMessage(d, c, true);
+		dropMessageRepository.addMessage(d, identity, c, true);
 		httpDropConnector.send(c, d);
 	}
 
 	void loadMessages(Contact c) throws EntityNotFoundExcepion {
```

Comment:
This is going to kill the client.
Imagine about 1k messages ... on each new message, every message is reloaded from the MessageRepository